src/models/getTutor.ts

```diff
@@ -0,0 +1,19 @@
+import zubiDB from "../database/db";
+
+// console.log(zubiDB);
+
+const tutors = zubiDB.prepare("SELECT * FROM tutors");
+
+const allRows = tutors.all();
+
+// const getTutorById = (index: number) => {
+//   return allRows[index - 1];
+// };
+
+const getTutorById = () => {
+  return allRows;
+};
+
+// console.log(getTutorById(2));
+
```

Comment:
Same here, console logs need to be removed.

Reply:
I've just noticed this is incomplete since you're still working on including a route that involves some sort of filtering. You, might want to look at creating a url query parameters and include that in tutorRoutes.